scripts/depsvalidator/Sources/DepsValidatorLibrary/Models/Process+Shell.swift

```diff
@@ -8,6 +8,7 @@ extension Process {
         if let environment = environment {
             process.environment = process.environment ?? [:]
             process.environment?.merge(environment, uniquingKeysWith: { (_, rhs) in rhs })
+            process.environment?.merge(ProcessInfo.processInfo.environment, uniquingKeysWith: { (_, rhs) in rhs })
```

Comment:
Not sure why this wasn't necessary before, but without this, custom rules were unable to inherit the PATH

Reply:
Should we run this merge before `process.environment?.merge(environment`? Wouldn't the ProcessInfo environment override variables?

Reply:
Yes, good catch